cps/kni.c

```diff
@@ -1860,7 +1860,7 @@ init_kni(const char *kni_kmod_path, unsigned int num_kni)
 		return -1;
 	}
 
-	ret = init_module(file, len, "");
+	ret = init_module(file, len, "carrier=on");
```

Comment:
I think that calling `rte_kni_update_link()` is safer than adding this parameter to the module because if the module is already loaded without this parameter, we'll have a weird behavior that will confuse network administrators and us while diagnosticating what's going on. I learned about `rte_kni_update_link()` in the link you posted in the description of this pull request.